sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClientOptions.cs

```diff
@@ -31,10 +31,10 @@ public FormRecognizerClientOptions(ServiceVersion version = LatestVersion)
         public enum ServiceVersion
         {
             /// <summary>
-            /// The V1 of the template service.
+            /// The V2.0
```

Comment:
`The V2.0 of the template service.`?

Reply:
hmm not sure... for me it sounds like template service comes from the template library we have in our repo ... so decided to remove

Reply:
I see. Maybe renaming:
```
The V2.0 of the service?
```

Regardless of our decision, we need to update the `ServiceVersion` description in the same way. Currently it is:
```
The template service version.
```

If you want to keep it as `The V2.0`, just don't forget to add the period `.` at the end.

Reply:
:( merged too fast.
I will add this in my next PR.
Thank you!